api/controllers/user.go

```diff
@@ -56,7 +56,7 @@ func (uc *UserController) GenerateToken(w http.ResponseWriter, r *http.Request)
 	err := json.NewDecoder(r.Body).Decode(&userLoginDto)
 
 	if err != nil {
-		fmt.Println(fmt.Errorf("#UserController.GenerateToken error: %w", err))
+		uc.logger.Error("error on user controller: ", err)
```

Comment:
Esse log tá em inglês mas os próximos que vi são em português algum motivo em específico?

Reply:
Eu havia me baseado na implementação de loggers de Ongs, agora to em dúvida qual usar

Reply:
os que estão em pt eram os originais, tentarei atualizar todos